src/svgGeometry.js

```diff
@@ -5,45 +5,57 @@
  */
 
 const path: {[string]: string} = {
+    // stdHorizRule is used for frac-lines, underlines, and overlines.
+    // It is 0.04em thick if the line comes from normalsize/textstyle.
+    // It is in a viewBox that is 5x as tall as the line, so that the
+    // full line will be rendered even if the browser rounds down
+    // the enclosing span size.
+    stdHorizRule: "M0 120 V80 H400000 v40 H0 z M0 120 V80 H400000 v40 H0 z",
```

Comment:
I assume all uses of `80` here for padding similar to what's in delimiter.js.  Maybe we should have a constant for these so that it's easy to update if necessary.

Reply:
Yup.